src/main/java/com/iotiq/application/service/ProductService.java

```diff
@@ -160,7 +166,7 @@ public byte[] exportCSVFile(UUID sellerId) throws IOException {
                     printer.printRecord(product.getTitle(), product.getDescription(), product.getPrice().getAmount(),
```

Comment:
Maybe we should rethink this `printRecord` function? This many arguments is excessive and error prone imo.

Reply:
@oguzIotiq

Reply:
Yes I think so, @ozdkaan please open an issue and we can refactor this there